### PR TITLE
Get Electron working again

### PR DESCRIPTION
**Pull request by itsjoeoui**
We can use Electron during development now!  Resolves STU-13

### CODE REVIEW

**Comment by yassine-moumen** [comment]
## To access Flojoy-studio build from this pull request, you can use the following URLs: 

Windows installation: http://588_windows.flojoy-test.com:3000 
Linux installation: http://588_linux.flojoy-test.com:3000 

Please note that it may take a few minutes for the instance to start up, so please be patient. 

It is important to know that the instance will automatically shut down after 4 hours to conserve resources. If you wish to start the instance again after it shuts down (or restart it to pickup new changes), you can use this API : 

Windows: https://kgpsoscpc4q52d4mtwuu7jhvie0qdiso.lambda-url.us-east-1.on.aws/start_instance?instance_id=i-0dc1117403883af85&domain=588_windows 

Linux: https://kgpsoscpc4q52d4mtwuu7jhvie0qdiso.lambda-url.us-east-1.on.aws/start_instance?instance_id=i-0334f1ade88272066&domain=588_linux

**Review by dev-nested — APPROVED** [review]

File: electron/main/index.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+// Here, you can also use other preload
+const preload = join(__dirname, "../preload/index.js");
+const url = process.env.VITE_DEV_SERVER_URL;
+const indexHtml = join(process.env.DIST, "index.html");
+
+async function createWindow() {
+  win = new BrowserWindow({
+    title: "Main window",
+    icon: join(process.env.PUBLIC, "favicon.ico"),
+    autoHideMenuBar: true,
```
Comment (smahmed776) [inline electron/main/index.ts:48]: This should be false for dev mode for debugging purpose and can set to true for packaged version IMO.

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Looks good

**Review by smahmed776 — APPROVED** [review]
looks good

**Pull request merged**